lib/govspeak.rb

```diff
@@ -292,7 +292,14 @@ def render_image(image)
     wrap_with_div("place", "$P", Govspeak::Document)
     wrap_with_div("information", "$I", Govspeak::Document)
     wrap_with_div("additional-information", "$AI")
-    wrap_with_div("example", "$E", Govspeak::Document)
+
+    extension("example", surrounded_by("$E")) do |body|
+      <<~BODY
+        <div class="example" markdown="1">
+          #{body.strip.gsub(/\A^\|/, "\n|").gsub(/\|$\Z/, "|\n")}
```

Comment:
**suggestion (if-minor):** documentation

Is this some kind of default? I can see that it's the same as what's done with CTAs, but it's different to how the address blocks code looks. Is there code or other documentation that could be linked to in this and maybe the next commit message to explain why:
1. moving from `wrap_with_div ... Govspeak::Document` to this RegEx is suitable here?
2. `markdown="1"` seems to be the key change for the abbreviation substitution to kick in (based on the next commit)